tensorflow/tools/ci_build/Dockerfile.rocm

```diff
@@ -0,0 +1,97 @@
+# This Dockerfile provides a starting point for a ROCm installation of 
+# MIOpen and tensorflow.  
+FROM ubuntu:xenial-20170619
```

Comment:
Why not a more generic image? "ubuntu:xenial"

Reply:
@parallelo Please weigh in. I believe the specific tag was specified to ensure the proper Linux kernel version is used?

Reply:
Correct.  We typically use a 4.13-45 linux kernel.

Reply:
Do we have an upper limit on the kernel version?
If we depend on one very specific kernel version this will be very very brittle.
Is there a way to lift this restriction?

Reply:
we had an issue with 4.15+ kernels in the older version of ROCm stack. Now the issue has been fixed in the upcoming ROCm release so I'll remove this.